bundle/manifests/test-nlb_v1_service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    aws-nlb-helper.3scale.net/enable-targetgroups-proxy-protocol: "true"
+    service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
+    service.beta.kubernetes.io/aws-load-balancer-type: nlb
+  creationTimestamp: null
+  labels:
+    app: aws-nlb-helper
+  name: test-nlb
```

Comment:
What is the purpose of this specific manifest, is it used at any place?

There is already the sample one at `config/samples/core_v1_service.yaml`, right?

Reply:
That is automatically generated by the bundling.

Reply:
Ok, it seems, as there is no CR, is taking the sample Service as a resource to deploy via `bundle`, instead of an olm example embedded in the CSV.

I would suggest to remove that service, otherwise a Service of type LoadBalancer  (AWS NLB) will be always created, incurring in costs ($$).

And instead, embed the example of annotated Service on the README in the CSV (like you already present que different annotations), so if anyone install it via OLM UI, would not see any CR (because there ware not), but will see the README with that annotated service example

Reply:
What about maybe adding the service example to README in the CSV? In the repor README service example is already added